.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+node_js: 
+  - "stable"
+cache:
+  directories:
+    - "node_modules"
+
+install: 
+  - npm install
+
+script: 
+  - npm test
```

Comment:
We should have a `before_script:`  here which would setup our test database, before running `npm test`

Reply:
Done